it/src/main/java/com/google/cloud/teleport/it/common/ResourceManagerUtils.java

```diff
@@ -35,7 +35,7 @@ public class ResourceManagerUtils {
 
   private static final int MIN_PROJECT_ID_LENGTH = 4;
   private static final int MAX_PROJECT_ID_LENGTH = 30;
-  private static final Pattern ILLEGAL_PROJECT_CHARS = Pattern.compile("[^a-zA-Z0-9-!']");
+  private static final Pattern ILLEGAL_PROJECT_CHARS = Pattern.compile("[^a-zA-Z0-9-!'-:]");
```

Comment:
Good, thanks! There's a `throw new IllegalArgumentException` below based on this, can you please adjust as well?